lib/scooter_service.dart

```diff
@@ -553,6 +553,24 @@ class ScooterService {
     _sendCommand("scooter:state unlock");
   }
 
+  Future<void> wakeUpAndUnlock() async {
+    await wakeUp();
+
+    // wait for stand-by and unlock
```

Comment:
In best we could listen until a certain value is present. I'd need to spend more time on this to figure it out. Eventually next iteration?